docker-compose.yml

```diff
@@ -37,8 +37,6 @@ services:
     image: postgres:13
     ports:
       - 5432:5432
-    env_file:
-      - db.env
```

Comment:
The removal of this seems to be causing issues with connecting to the database, at least when testing locally

Reply:
Whoops! Good catch. Didn't mean to remove that